apps/arena/lib/arena/configuration.ex

```diff
@@ -7,14 +7,41 @@ defmodule Arena.Configuration do
     Enum.find(config.characters, fn character -> character.name == name end)
   end
 
-  def get_game_config() do
+  def get_game_config(map_selected \\ "") do
     client_config = get_client_config()
 
-    get_current_game_configuration()
+    get_current_game_configuration(map_selected)
     |> Map.put(:client_config, client_config)
   end
 
-  defp get_current_game_configuration do
+  def get_game_mode_configuration(name, type) do
+    gateway_url = Application.get_env(:arena, :gateway_url)
+    query_params = URI.encode_query(%{"name" => name, "type" => type})
```

Comment:
This kind of query is very error-prone. Type should be enough.
If we need something to differentiate between teams and solo, maybe we could create another field in the game mode schema as an enum for team mode types (:solo, :duo, :trio, etc).